docs/installation.md

```diff
@@ -154,24 +154,27 @@ closer to what you'd run in production.
 EventStoreDB has a Docker image available for any platform that supports Docker.
 
 The following command will start the EventStoreDB node using default HTTP port, without security. You can then
-connect to it using one of the clients and the `esdb://localhost:2113?tls=false` connection string. The Admin
-UI will be accessible, but the Stream Browser won't work (as it needs AtomPub to be enabled).
+connect to it using one of the clients and the `esdb://localhost:2113?tls=false` connection string. 
```

Comment:
Please add "as well as open the Admin UI by opening http://localhost:2113 in your browser"